pyramid/config/views.py

```diff
@@ -1627,20 +1627,25 @@ def notfound(request):
         Pyramid will return the result of the view callable provided as
         ``view``, as normal.
 
+        :class:`~pyramid.httpexceptions.HTTPTemporaryRedirect` class is used
```

Comment:
This paragraph feels weird, it should likely be documented under the `AppendSlashNotFoundViewFactory`.

Or maybe it should reference the fact that this is related to `append_slash`. Maybe remove the newline and continue the paragraph so it seems related? Not sure, but this currently reads weird to me.

Reply:
Just a little further down, in the next paragraph we already explain which default we are using... maybe add the info about redirecting with the same HTTP method as a comment there?

Reply:
I didn't want to document it in `AppendSlashNotFoundViewFactory` since it's deprecated and I'd imagine 99% folks will be looking at docstring of `add_notfound_view`.

I'll move it below the next paragraph :)